### PR TITLE
Edit place functionality

### PR DESCRIPTION
**Pull request by RoxVer**
Method for editing place + unit test

### CODE REVIEW

File: src/main/webapp/resources/app/routes.js [hunk 1]
```diff
@@ -9,5 +9,7 @@ angular.module('greenApp').config(function($routeProvider) {
 		template : '<user></user>'
 	}).when('/map', {
 		template : '<map></map>'
+	}).when('/place/place/:placeId', {
```
Comment (Sabfir) [inline src/main/webapp/resources/app/routes.js]: It seems like too many "place" here

Comment (RoxVer) [reply src/main/webapp/resources/app/routes.js]: Fixed

File: src/main/webapp/resources/app/css/place.css [hunk 2]
```diff
@@ -0,0 +1,79 @@
...
+	margin: 0;	
+} 
+*/
+
+/*
+img.responsive-img {
+    max-width: 35%;
+    height: auto;
+}
+*/
```
Comment (ghost) [inline src/main/webapp/resources/app/css/place.css]: We decided to have only one css file.
You should move this part in style.css with mark before something like /_-------------- Place css  --------------_/

**Comment by 47souls** [comment]
Reviewed.

**Comment by ghost** [comment]
Reviewed

**Pull request merged**